test/WorkerExtensionTests/Blob/BlobStorageConverterTests.cs

```diff
@@ -22,375 +24,773 @@
 using Moq;
 using Xunit;
 
-using Constants = Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs.Constants;
-
+// Tests missing for BlobBaseClient; BlockBlobClient; PageBlobClient; AppendBlobClient
+// Moq does not support mocking extension methods directly
```

Comment:
@fabiocav open to suggestions on how to address this. We could always introduce a factory but I figured that might be overkill for right now? Thoughts?